apps/hash-api/src/graph/index.ts

```diff
@@ -47,6 +48,52 @@ export const zeroedGraphResolveDepths: GraphResolveDepths = {
   hasRightEntity: { incoming: 0, outgoing: 0 },
 };
 
+export const currentTemporalAxes: QueryTemporalAxesUnresolved = {
+  pinned: {
+    axis: "transactionTime",
+    timestamp: null,
+  },
+  variable: {
+    axis: "decisionTime",
+    interval: {
+      start: null,
+      end: null,
+    },
+  },
+};
+
+export const fullDecisionTimeAxis: QueryTemporalAxesUnresolved = {
+  pinned: {
+    axis: "transactionTime",
+    timestamp: null,
+  },
+  variable: {
+    axis: "decisionTime",
+    interval: {
+      start: {
+        kind: "unbounded",
+      },
+      end: null,
+    },
+  },
+};
+
+export const fullTransactionTimeAxis: QueryTemporalAxesUnresolved = {
```

Comment:
Is this used anywhere? if it's not I'd suggest we remove it

Reply:
It's not used, yet, but I'm going to use it in the tests I'm about to write. (stated that they are unused in the OP)